specter-registry/src/client/java/dev/spiritstudios/specter/impl/registry/SpecterRegistryClient.java

```diff
@@ -1,18 +1,34 @@
 package dev.spiritstudios.specter.impl.registry;
 
+import com.mojang.serialization.Lifecycle;
 import dev.spiritstudios.specter.api.registry.metatag.Metatag;
+import dev.spiritstudios.specter.api.registry.reloadable.ClientReloadableRegistryEvents;
 import dev.spiritstudios.specter.impl.registry.metatag.MetatagValueHolder;
 import dev.spiritstudios.specter.impl.registry.metatag.data.MetatagReloader;
 import dev.spiritstudios.specter.impl.registry.metatag.network.MetatagSyncS2CPayload;
+import dev.spiritstudios.specter.impl.registry.reloadable.SpecterReloadableRegistriesImpl;
+import dev.spiritstudios.specter.impl.registry.reloadable.network.ReloadableRegistrySyncS2CPayload;
 import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.client.networking.v1.ClientPlayConnectionEvents;
 import net.fabricmc.fabric.api.client.networking.v1.ClientPlayNetworking;
 import net.fabricmc.fabric.api.resource.ResourceManagerHelper;
 import net.minecraft.client.MinecraftClient;
-import net.minecraft.registry.Registry;
+import net.minecraft.client.network.ClientPlayNetworkHandler;
+import net.minecraft.registry.*;
```

Comment:
You should configure your IDE to disable star imports. they are automatically handled anyway, and makes it harder for people to see where classes come from when looking at code outside an IDE (such as during code review)